include/sentry.h

```diff
@@ -650,6 +651,11 @@ SENTRY_API sentry_user_consent_t sentry_user_consent_get(void);
  */
 SENTRY_API sentry_uuid_t sentry_capture_event(sentry_value_t event);
 
+/*
+ * Captures an exception to be handled by the backend.
```

Comment:
Like I mentioned in chat, it would be good to better document the expectations for this function.